### PR TITLE
Fixes for how to guides

### LINKED ISSUE #1588: Fixes in How To guides
There are some how to guides which require several small changes for broken links, etc. This issue will track progress of fixing those issues.

### PR DESCRIPTION
**Pull request by kessler-frost**
- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

Resolves #1588 and #1585 along with minor fixes to some other RTD files.

### CODE REVIEW

File: doc/source/how_to/execution/covalent_cli.rst [hunk 1]
```diff
@@ -94,4 +94,10 @@ Reset the configuration using the :code:`purge` subcommand:
 .. code-block:: sh
 
     $ covalent purge
-    Covalent server files have been purged.
+    ********************* WARNING *********************
+    Purging will perform the following operations:
+    1. Stop the covalent server if running.
+    2. /home/neptune/.cache/covalent directory will be deleted.
```
Comment (FyzHsn) [inline doc/source/how_to/execution/covalent_cli.rst:100]:
```suggestion
    2. /home/user/.cache/covalent directory will be deleted.
```

File: doc/source/how_to/execution/covalent_cli.rst [hunk 2]
```diff
@@ -94,4 +94,10 @@ Reset the configuration using the :code:`purge` subcommand:
...
+    3. /home/neptune/.config/covalent directory will be deleted.
```
Comment (FyzHsn) [inline doc/source/how_to/execution/covalent_cli.rst:101]:
```suggestion
    3. /home/user/.config/covalent directory will be deleted.
```

File: doc/source/how_to/coding/add_constraints_to_lattice.ipynb [hunk 3]
```diff
@@ -22,24 +22,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": null,
```
Comment (FyzHsn) [inline doc/source/how_to/coding/add_constraints_to_lattice.ipynb:25]: The cells need to be executed.

**Review by FyzHsn — CHANGES_REQUESTED** [review]
- Also, dispatch cancellation link is missing in here: https://covalent--1596.org.readthedocs.build/en/1596/api/

File: doc/source/how_to/coding/add_constraints_to_lattice.ipynb (hunk 3 above)
Comment (kessler-frost) [reply doc/source/how_to/coding/add_constraints_to_lattice.ipynb:25]: Even after executing all the cells in the notebook, this variable still seems to be null for me.

**Review by FyzHsn — APPROVED** [review]

**Review by santoshkumarradha — APPROVED** [review]

**Pull request merged**